### DISCUSSION TITLE
WebUI

### DISCUSSION THREAD

**Discussion by MydsiIversen** [gorilla-devs/ferium#444] [Q&A]
Hello! Is there any kind of WebUI for this project? Or is it planned?

**Comment by theRookieCoder** [comment]
I did initially have the idea of a GUI for ferium, not necessarily using WebUI. But I realised there are already great GUI mod managers built into many modern launchers. I decided ferium should remain in the CLI niche rather than try to expand into conquered territory.

There isn't any graphical interface to my knowledge, even third party ones.

**Comment by MydsiIversen** [comment]
I think it would fit a web ui though to generate server/client modpacks, I have made some designs to it. And are going to make a prototype this week see if the POC would be good.

Fra: Ilesh Thiada ***@***.***>
Sendt: 17. november 2024 10:48
Til: gorilla-devs/ferium ***@***.***>
Cc: Mads Iversen (MADS51R1) ***@***.***>; Author ***@***.***>
Emne: Re: [gorilla-devs/ferium] WebUI (Discussion #444)

You don't often get email from ***@***.******@***.***>. Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

I did initially have the idea of a GUI for ferium, not necessarily using WebUI. But I realised there are already great GUI mod managers built into many modern launchers. I decided ferium should remain in the CLI niche rather than try to expand into conquered territory.

**Reply by theRookieCoder** [reply]
> generate server/client modpacks

If you are planning on exporting ferium profiles to modpacks, I'll point you to the https://github.com/gorilla-devs/ferium/tree/export branch.

**Comment by MydsiIversen** [comment]
Is it because it’s more stable or?

Fra: Ilesh Thiada ***@***.***>
Sendt: 18. november 2024 16:32
Til: gorilla-devs/ferium ***@***.***>
Cc: Mads Iversen (MADS51R1) ***@***.***>; Author ***@***.***>
Emne: Re: [gorilla-devs/ferium] WebUI (Discussion #444)

You don't often get email from ***@***.******@***.***>. Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

generate server/client modpacks

If you are planning on exporting ferium profiles to modpacks, I'll point you to the https://github.com/gorilla-devs/ferium/tree/export branch.

**Reply by theRookieCoder** [reply]
I had a working implementation of exporting, and I even merged it (which is why github shows that it has no commits ahead of main), but the feature was too complex for me to include in the next update and I reverted it. The commit the branch is on still has the implementation (maybe I should remove the branch and just tag that commit).
